src/shogun/modelselection/GradientModelSelection.cpp

```diff
@@ -219,16 +219,16 @@ CParameterCombination* CGradientModelSelection::select_model(bool print_state)
 		nlopt_opt opt=nlopt_create(NLOPT_LD_MMA, total_variables);
 
 		// create lower bound vector (lb=-inf)
-		SGVector<double> lower_bound(total_variables);
-		lower_bound.set_const(1e-6);
+		//SGVector<double> lower_bound(total_variables);
```

Comment:
once positive parameters are moved into log domain.
this is a unconstrained problem.